pebblo/app/service/loader/document/document.py

```diff
@@ -123,7 +129,6 @@ def _update_document(document, snippet):
     def create_or_update_document(self, app_loader_details, data_source):
```

Comment:
Can we add type in function params and output type

Reply:
Also comments will be helpful

Reply:
Fixed.